### ISSUE TITLE
[Bug]: Settings inaccessible

### ISSUE THREAD

**Issue by Dr-Escher** [bottlesdevs/Bottles#3594]
### Describe the bug

Settings page of a bottle is inaccessible using latest version

### To Reproduce

1. Update to Bottles 51.16
2. Select some bottle
3. Click "Settings (configure Bottle settings.)"
4. Observe blank page

### Package

Flatpak from Flathub

### Distribution

Manjaro KDE Plasma

### Debugging Information

```shell
Official Package: true
Version: '51.16'
DE/WM: plasma
Display:
    X.org: true
    X.org (port): :0
    Wayland: false
Graphics:
    vendors:
        amd:
            vendor: amd
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/radeon_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/radeon_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 6.6.63-1-MANJARO
Disk:
    Total: 16672632832
    Free: 16672452608
RAM:
    MemTotal: 31.1GiB
    MemAvailable: 24.3GiB
Bottles_envs: null
```

### Troubleshooting Logs

```shell
(process:2): Gtk-WARNING **: 07:19:19.762: Locale not supported by C library.
        Using the fallback 'C' locale.

(bottles:2): Gtk-WARNING **: 07:19:20.155: Theme parser warning: style.css:184:3-185:1: Expected ';' at end of block

(bottles:2): Gtk-WARNING **: 07:19:20.155: Theme parser warning: style-dark.css:23:3-24:1: Expected ';' at end of block
07:19:20 (INFO) Launching with URI: None 
07:19:20 (INFO) [Activate] request received. 
07:19:20 (INFO) Bottles Started! 
07:19:28 (INFO) Performing Bottles checks… 
07:19:28 (INFO) Dxvks found:
         - x32
         - x64
         - dxvk-2.5.1
 
07:19:28 (INFO) Runners found:
         - soda-9.0-1
         - sys-wine-9.0
 
07:19:28 (INFO) Temp directory cleaned successfully! 
07:19:28 (INFO) Bottles found:
         - Bottle
 
07:19:28 (INFO) Catalog installers loaded 
07:19:28 (INFO) Catalog dependencies loaded 
07:19:28 (INFO) Catalog components loaded 
Traceback (most recent call last):
  File "/app/share/bottles/bottles/frontend/views/list.py", line 144, in show_details
    self.window.show_details_view(config=config)
  File "/app/share/bottles/bottles/frontend/windows/main_window.py", line 323, in show_details_view
    self.page_details.set_config(config or BottleConfig())
  File "/app/share/bottles/bottles/frontend/views/details.py", line 201, in set_config
    self.view_preferences.set_config(config=config)
  File "/app/share/bottles/bottles/frontend/views/bottle_preferences.py", line 478, in set_config
    self.combo_nvapi.set_selected(self.manager.nvapi_available.index(self.config.NVAPI))
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: 'dxvk-nvapi-v0.5.4' is not in list

(bottles:2): Gtk-WARNING **: 07:19:35.102: Child name 'preferences' not found in GtkStack
```

### Additional context

Last error message log line appears on 'Settings' click

**Comment by TheEvilSkeleton** [comment]
> happening to me too. @TheEvilSkeleton @mirkobrombin @jntesteves is a major regression. how did it pass testing? rolling back to previous version of Bottles (51.15) fixes the issue

I didn't encounter that bug when I tested it locally

**Comment by Leniptic** [comment]
For me as well. Along with desktop shortcuts not working anymore. Considering how often I need the settings menu, half the app is practically unusable right now lol

**Comment by Schildkroet** [comment]
Hello,
i am experiencing the same problem.

**Comment by TheEvilSkeleton** [comment]
> Important checks were removed... [af29960#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L497](https://github.com/bottlesdevs/Bottles/commit/af2996044bfdf55de45b187764495e1f6cfedd09#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L497) [af29960#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L505](https://github.com/bottlesdevs/Bottles/commit/af2996044bfdf55de45b187764495e1f6cfedd09#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L505) [af29960#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L513](https://github.com/bottlesdevs/Bottles/commit/af2996044bfdf55de45b187764495e1f6cfedd09#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L513) [af29960#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L520](https://github.com/bottlesdevs/Bottles/commit/af2996044bfdf55de45b187764495e1f6cfedd09#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L520) [af29960#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L526](https://github.com/bottlesdevs/Bottles/commit/af2996044bfdf55de45b187764495e1f6cfedd09#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L526) [af29960#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L548](https://github.com/bottlesdevs/Bottles/commit/af2996044bfdf55de45b187764495e1f6cfedd09#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L548) The ideal is to always check if the value exists in the array/list before trying to get its index with the `.index()` method, or you'll get a ValueError.

I think the bigger problem is that it's trying to load things that aren't installed.

**Comment by Schildkroet** [comment]
> A possible workaround involves downloading some `DXVK-NVAPI` from the menu (I installed the latest `dxvk-nvapi-v0.7.1`) and then proceeded to open `bottle.yml` and set `NVAPI: dxvk-nvapi-v0.7.1` manually. Now the Setting open again. I can't find any reference to NVAPI there tho, maybe it's obsolete.

I had the exact same problem. But why was NVAPI set in my case? I have an AMD and dont even use it.

**Comment by Vingian** [comment]
> > Important checks were removed... [af29960#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L497](https://github.com/bottlesdevs/Bottles/commit/af2996044bfdf55de45b187764495e1f6cfedd09#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L497) [af29960#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L505](https://github.com/bottlesdevs/Bottles/commit/af2996044bfdf55de45b187764495e1f6cfedd09#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L505) [af29960#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L513](https://github.com/bottlesdevs/Bottles/commit/af2996044bfdf55de45b187764495e1f6cfedd09#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L513) [af29960#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L520](https://github.com/bottlesdevs/Bottles/commit/af2996044bfdf55de45b187764495e1f6cfedd09#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L520) [af29960#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L526](https://github.com/bottlesdevs/Bottles/commit/af2996044bfdf55de45b187764495e1f6cfedd09#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L526) [af29960#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L548](https://github.com/bottlesdevs/Bottles/commit/af2996044bfdf55de45b187764495e1f6cfedd09#diff-a291ab81da17f87b97e189ad7058354a51882be0c90ae164793fd237b41ef877L548) The ideal is to always check if the value exists in the array/list before trying to get its index with the `.index()` method, or you'll get a ValueError.
> 
> I think the bigger problem is that it's trying to load things that aren't installed.

In my opinion... that's not the problem, as things like this can happen for various reasons(from corrupted files to moved bottle folders)... and the settings screen needs to be accessible regardless of anything(so... any error that could occur needs to be prevented).

**Comment by TheEvilSkeleton** [comment]
Emphasis on "the bigger problem". I apologize for being unclear; I didn't mean to dismiss the problem, rather I was stating that it's naively picking values when it shouldn't

**Comment by TheEvilSkeleton** [comment]
Closed by https://github.com/bottlesdevs/Bottles/pull/3600

**Comment by SjoerdV** [comment]
> > Closed by #3600
> 
> 
> 
> @TheEvilSkeleton @mirkobrombin thank you for fixing this!

I still see blank screens in two Global Preferences tabs (the ones for managing runners and dll components)
Present in .16 and .17
Rolling back to .15 fixes this.

Maybe a 51.18 release is in order?

**Comment by RedSnt** [comment]
I'm not sure what I've done to make this happen in 51.18, but yeah, downgrading to 51.15 did the trick. Unfortunately now unable to get the latest runners or DLL components.
